.github/workflows/pull-db_test.yml

```diff
@@ -204,18 +188,14 @@ jobs:
         ports:
           - "3306:3306"
     steps:
-      - name: checkout
-        uses: actions/checkout@v3
-      - name: setup go
-        uses: actions/setup-go@v4
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
         with:
           go-version: '>=1.20.0'
       - name: Add hosts to /etc/hosts
         run: echo "127.0.0.1 mysql8" | sudo tee -a /etc/hosts
-      - name: install dependencies
-        run: make deps-backend
-      - name: build
-        run: make backend
+      - run: make deps-backend
+      - run: make backend
         env:
           GOPROXY: https://goproxy.io
```

Comment:
We could probably remove the "GOPROXY"/"GOSUMDB" envs. I had them included to keep things close to 1:1, but it might trim things down a bit.

Reply:
Where are the runners running? Is there any chance for runners to run in China?

Reply:
Ok, will remove those.

Reply:
done